edith-step/src/ReceiptInput.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shallow, configure, mount } from 'enzyme';
+import ReceiptInput from './ReceiptInput';
+import './setupTests.js'
+
+// Submit button calls handleSubmit when clicked
+it('should call handleSumbit when Submit button is clicked', () => {
+  const handleSubmit = jest.spyOn(ReceiptInput.prototype, 'handleSubmit');
+  const component = mount(<ReceiptInput onSubmit={ handleSubmit }/>);
+
+  component.find('form').simulate('submit');
+  expect(handleSubmit).toBeCalled();
+  component.unmount();
+});
+
+// handleSubmit resets state
```

Comment:
niit: add period at the end of the comment.

https://jestjs.io/docs/en/setup-teardown#repeating-setup-for-many-tests